Test/Framework/Graphics/SpriteBatchTest.cs

```diff
@@ -296,7 +294,7 @@ public void DrawWithTexture()
         [Test]
         public void DrawWithCustomEffectAndTwoTextures()
         {
-            var customSpriteEffect = AssetTestUtility.CompileEffect(gd, "CustomSpriteBatchEffect.fx");
+            var customSpriteEffect = content.Load<Effect>(Paths.Effect("CustomSpriteBatchEffect"));
```

Comment:
When you get the runtime going and tests are running on Windows, I think it should do the same as DirectX and build the effect.  For now, I think the above should be ok.